### DISCUSSION TITLE
Problem syncing clock with Google

### DISCUSSION THREAD

**Discussion by EstevaoTerci** [Authenticator-Extension/Authenticator#789] [Q&A]
I am unable to sync the clock with Google on some machines. I currently use the extension on a few different machines and on some of them, when I try to sync the extension's internal clock with Google, the extension returns an error saying "Caution! Your local clock has been stopped for a long time, please adjust it before Continue." However, the local clock is correct and configured for automatic synchronization, just like on other machines where the extension works perfectly. I'm currently using updated Chrome, on machines running windows 7.

![image](https://user-images.githubusercontent.com/40184342/141304033-ae9646d2-9509-4f12-aaba-e9edb48a8993.png)

**Comment by Vadym-Semchyshyn** [comment]
I am facing same issue. Please help me out.
